src/tunacode/tools/ignore.py

```
@@ -170,6 +171,28 @@ def get_ignore_manager(root: Path) -> IgnoreManager:
     return ignore_manager
 
 
+def traverse_gitignore(
+    entry: DirEntry, ignore_manager: IgnoreManager, recursive: bool, stack: list[Path]
```

Ruff (0.14.14)

Boolean-typed positional argument in function definition

(FBT001)